src/library/librarycontrol.cpp

```diff
@@ -482,16 +482,26 @@ void LibraryControl::emitKeyEvent(QKeyEvent&& event) {
     VERIFY_OR_DEBUG_ASSERT(m_pLibraryWidget) {
         return;
     }
-    if (!m_pLibraryWidget->hasFocus() && !m_pSidebarWidget->hasFocus()) {
+    if (!QApplication::focusWindow()) {
+        qDebug()<<"Mixxx window is not focused, don't send key events";
```

Comment:
Is this worth a debug message after the fix is merged?

Reply:
Do you want to return here?

Reply:
`return;` added